features/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/publisher/themes/default/js/logic/lifecycle.asset.js

```diff
@@ -159,7 +159,7 @@ $(function() {
 					 graph.Renderer.initRaphael();
 					 graph.Renderer.render(graph.NMap);
 					 }       */
-					var statInfo = JSON.parse(response);
+					var statInfo = response;
```

Comment:
removed redundant variable `statInfo` in commit 30deae1c9110153593e9e0db62fd72e81fccc44e of PR #240